packages/server/src/electron.js

```diff
@@ -18,11 +19,18 @@ async function startApp() {
   delete require.cache[require.resolve("./environment")]
   require("dotenv").config({ path: envFile })
 
-  if (isDev) {
-    unhandled({
-      showDialog: true,
-    })
-  }
+  unhandled({
+    showDialog: true,
+    reportButton: error => {
+      openNewGitHubIssue({
```

Comment:
NAB: should we add a title? Having a pre-set title might make it easier to filter these automated crash reports from other issues people have reported. I know they could always change the title but it might help.

Reply:
Or what about setting an `errorReport` label with the `labels` property? Just so that we can filter them down with those labels?

Reply:
Sounds great - good suggestions. Will update the title using the message from the error and add an `error-report` label also 👍